website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -14,10 +14,11 @@ This document explains the security model of Terraform Cloud and the security co
 
 ## Important Concepts
 
-### Workspaces and Teams
-
+### Projects, Workspaces, and Teams
 Terraform Cloud organizes infrastructure with workspaces. Workspaces represent a logical security boundary within the organization. Variables, state, SSH keys, and log output are local to a workspace. You can grant teams [read, plan, write, admin, or a customized set of permissions](/cloud-docs/users-teams-organizations/permissions) within a workspace.
 
+Projects let you group  related workspaces in your organization. You can use projects to assign read or admin permissions to a particular team across all  workspaces in the project.
```

Comment:
```suggestion
Projects let you group related workspaces in your organization. You can use projects to assign read or admin permissions to a particular team across all workspaces in the project.
```

Reply:
Changes made in this PR to be merged upon approval: https://github.com/hashicorp/terraform-docs-common/pull/245